autonomx/main.cpp

```diff
@@ -51,9 +51,13 @@ int main(int argc, char *argv[]) {
     #endif
 
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
-
     QGuiApplication app(argc, argv);
 
+    // global settings
+    QCoreApplication::setApplicationName("autonomX");
+    QCoreApplication::setApplicationVersion("0.1.1");
```

Comment:
The software version number should be in a unique constant, if possible.

Reply:
@aalex AFAIK, the call at line 58 does define the unique constant; it's used in QML as `Qt.application.version`, so this is really the only place where we ever declare its value statically. i could always do a separate const variable right before and assign that to `setApplicationVersion`, but i don't really see the point of adding an extra variable if we're using it only once... unless i'm misunderstanding the request here?

Reply:
@netherwaves Yes: I think using a constant will make the code easier to read and maintain. Even if this variable is used only at one place in the source code, it still adds value to the project. It makes it easier for the release manager to know where to change the version number.

Reply:
Actually, this is how it was before, since I had set it like that.